### PR TITLE
Adding properties for H-level functionality

### PR DESCRIPTION
**Pull request by IanRay11**

### CODE REVIEW

**Review by martinescardo — APPROVED** [review]
Thanks. I'll merge when the tests pass.

File: source/UF/Singleton-Properties.lagda [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+Σ-is-singleton : {X : 𝓤 ̇ } {A : X → 𝓥 ̇ }
+               → is-singleton X
+               → ((x : X) → is-singleton (A x))
+               → is-singleton (Σ A)
+Σ-is-singleton {X = X} {A = A} (c , C) h = ((c , center (h c)) , Σ-centrality)
+ where
+  Σ-centrality : is-central (Σ A) (c , center (h c))
+  Σ-centrality (x , a) = ⌜ Σ-＝-≃ ⌝⁻¹ (C x , p)
+   where
+    p : transport A (C x) (center (h c)) ＝ a
```
Comment (martinescardo) [inline source/UF/Singleton-Properties.lagda:38]: The type of `p` could be omitted because it is clear from the equational reasoning what it is.

**Comment by IanRay11** [comment]
One second. I forgot to add it to the index. @martinescardo

**Comment by martinescardo** [comment]
OK. I'll wait.

**Review by martinescardo — APPROVED** [review]

**Pull request merged**